bootstrap.sh

```diff
@@ -4,6 +4,20 @@
 #/ Heavily inspired by https://github.com/MikeMcQuaid/strap
 set -e
 
+# Default cloning value.
+GIT_URL_PREFIX="git@github.com:"
+
+if [ -n "$STRAP_CI" ]; then
+  echo "Running within CI..."
+  CODE_ROOT="$HOME/code"
+  SKIP_METRICS=1
+  GIT_URL_PREFIX="https://github.com/"
+  SKIP_GETSENTRY=1
```

Comment:
This workflow will not be checking out getsentry.
I'm OK to not invest a lot more energy to being able to checkout getsentry. In fact, I would be concerned since this repo is public.